extensions/windows-event-log/ConsumeWindowsEventLog.cpp

```diff
@@ -134,8 +135,8 @@ core::Property ConsumeWindowsEventLog::OutputFormat(
   core::PropertyBuilder::createProperty("Output Format")->
   isRequired(true)->
   withDefaultValue(Both)->
-  withAllowableValues<std::string>({XML, Plaintext, Both})->
-  withDescription("Set the output format type. In case \'Both\' is selected the processor generates two flow files for every event captured")->
+  withAllowableValues<std::string>({XML, Plaintext, Both, JSONSimple, JSONFlattened, JSONRaw})->
```

Comment:
this idea came up during design and having discussed it with @arpadboda, the conclusion was that a single property is simpler than a separate json-specific property on the CWEL

Reply:
Even when we have list-type properties, we could have "XML,JSON" or "JSON,Plaintext" as the output format property. A user would understand what "JSON" is; they would not know (without asking us or the support team) what JSONRaw, JSONSimple, JSONFlattened mean and how to choose between them.

I understand you have had this discussion already and have made a decision, but I think it would be worth discussing it some more, as this decision causes our code to be more complex and the user interface more difficult to use, both meaning more work in the future.

Reply:
my personal preference is JSON to mean JSONRaw and a template-based json transformer processor (jolt processor in nifi), but based on simplicity and possible time-constraints that might not be viable

Reply:
the format has been separated to JSON + flavor in a new property `JSON Format`